ui/pages/confirmations/components/confirm/title/title.tsx

```diff
@@ -72,6 +74,9 @@ const getTitle = (
       if (isNFT) {
         return t('confirmTitleApproveTransaction');
       }
+      if (customSpendingCap === '0') {
```

Comment:
Does this imply that the tile of the confirmation changes dynamically as the spending cap is changed?

Do we definitely want that?